pkgs/sdk-pkg/src/genie/libs/sdk/apis/tests/iosxe/eigrp/verify/verify_eigrp_interfaces_timers/mock_data/iosxe/mock_data.yaml

```
@@ -0,0 +1,57 @@
+configure:
+  commands:
+    end:
+      new_state: execute
+    line console 0:
+      new_state: configure_line
+    no logging console: ''
+  prompt: R1(config)#
+configure_line:
+  commands:
+    end:
+      new_state: execute
+    exec-timeout 0: ''
+  prompt: R1(config-line)#
+connect:
+  commands:
+    ? ''
+    : new_state: execute
+  preface: 'Trying mock_device ...
+
+    Connected to mock_device.
+
+    Escape character is ''^]''.'
+  prompt: ''
+execute:
+  commands:
+    config term:
+      new_state: configure
+    config-transaction:
+      new_state: configure
+    show ip eigrp interfaces detail:
+      response:
+      - "EIGRP-IPv4 Interfaces for AS(1)\r\n                              Xmit Queue\
+        \   PeerQ        Mean   Pacing Time   Multicast    Pending\r\nInterface  \
+        \            Peers  Un/Reliable  Un/Reliable  SRTT   Un/Reliable   Flow Timer\
+        \   Routes\r\nFa0/0                    1        0/0       0/0          32\
+        \       0/0           50           0\r\n  Hello-interval is 5, Hold-time is\
+        \ 15\r\n  Split-horizon is enabled\r\n  Next xmit serial <none>\r\n  Packetized\
+        \ sent/expedited: 1/0\r\n  Hello's sent/expedited: 303/2\r\n  Un/reliable\
+        \ mcasts: 0/1  Un/reliable ucasts: 1/2\r\n  Mcast exceptions: 0  CR packets:\
+        \ 0  ACKs suppressed: 0\r\n  Retransmissions sent: 1  Out-of-sequence rcvd:\
+        \ 1\r\n  Topology-ids on interface - 0 \r\n  Authentication mode is not set\r\
+        \nEIGRP-IPv4 Interfaces for AS(2)\r\n                              Xmit Queue\
+        \   PeerQ        Mean   Pacing Time   Multicast    Pending\r\nInterface  \
+        \            Peers  Un/Reliable  Un/Reliable  SRTT   Un/Reliable   Flow Timer\
+        \   Routes\r\nFa1/0                    0        0/0       0/0           0\
+        \       0/0            0           0\r\n  Hello-interval is 5, Hold-time is\
+        \ 15\r\n  Split-horizon is enabled\r\n  Next xmit serial <none>\r\n  Packetized\
+        \ sent/expedited: 0/0\r\n  Hello's sent/expedited: 304/1\r\n  Un/reliable\
+        \ mcasts: 0/0  Un/reliable ucasts: 0/0\r\n  Mcast exceptions: 0  CR packets:\
+        \ 0  ACKs suppressed: 0\r\n  Retransmissions sent: 0  Out-of-sequence rcvd:\
+        \ 0\r\n  Topology-ids on interface - 0 \r\n  Authentication mode is not set"
+      response_type: circular
+    show version: ''
```

please add this line as it might be called due to some changes in device.connect()

```suggestion
    show version: ''
    show version | include operating mode: ''
```

---

I added the show version | include operating mode: '' line